server/modules/nodeDef/manager/nodeDefManager.js

```diff
@@ -88,14 +88,15 @@ const updateNodeDefProps = async (user, surveyId, nodeDefUuid, props, propsAdvan
       ? await _updateNodeDefOnCyclesUpdate(surveyId, nodeDefUuid, props[NodeDef.propKeys.cycles], t)
       : []
 
+    const nodeDef = await NodeDefRepository.updateNodeDefProps(surveyId, nodeDefUuid, props, propsAdvanced, t)
```

Comment:
any reason to have moved this outside the Promise.all?

Reply:
no... I'll fix it